src/commons/utils/Constants.ts

```diff
@@ -77,6 +77,18 @@ for (let i = 1; ; ++i) {
   authProviders.set(id, { name, endpoint, isDefault: false, type: AuthProviderType.CAS });
 }
 
+for (let i = 1; ; ++i) {
+  const id = process.env[`REACT_APP_SAML_PROVIDER${i}`];
+  if (!id) {
+    break;
+  }
+
+  const name = process.env[`REACT_APP_SAML_PROVIDER${i}_NAME`] || 'Unnamed provider';
+  const endpoint = process.env[`REACT_APP_SAML_PROVIDER${i}_ENDPOINT`] || '';
+
+  authProviders.set(id, { name, endpoint, isDefault: false, type: AuthProviderType.SAML_SSO });
+}
```

Comment:
maybe we can consider abstracting out this logic, since its repetitive

Reply:
Hmmm, after considering it, the special case of `i === 1` being the default auth provider only for OAuth makes the abstraction unnecessarily slightly less trivial. Might be worth investigating if there is an actual need to have a "default" auth provider.

I think we can just leave it for now; we already have brotli compression enabled for our bundle in deployment so should not make much of a difference.